acceptance/tests/basic/basic_test.go

```diff
@@ -50,6 +50,7 @@ func TestBasicInstallation(t *testing.T) {
 				"global.tls.enabled":                   strconv.FormatBool(c.secure),
 				"global.gossipEncryption.autoGenerate": strconv.FormatBool(c.secure),
 				"global.tls.enableAutoEncrypt":         strconv.FormatBool(c.autoEncrypt),
+				"client.enabled":                       "true",
```

Comment:
Clients are now disabled by default but still supported. We can use this test to check that they still work.